web-ui/src/main/resources/catalog/js/edit/EditorController.js

```diff
@@ -237,7 +240,7 @@
                 $scope.layout.hideTopToolBar = true;
 
                 angular.extend(gnCurrentEdit, {
-                  id: gnCurrentEdit.metadata.id,
+                  id: '00000000000000000000' + gnCurrentEdit.metadata.id,
```

Comment:
it looks like this id is used at least for the placement of the view menu (which is in a different place in directories and records editing).

Before
![image](https://user-images.githubusercontent.com/1701393/152482369-3b64b8e0-245d-4bf1-8d7e-7f1ff0947ac5.png)

After
![image](https://user-images.githubusercontent.com/1701393/152482229-1e59d253-6349-41a5-be4b-6c286d8f6425.png)

Check in setViewMenuInTopToolbar ?

Reply:
how could that be even related?

Reply:
The id is used in various places 

![image](https://user-images.githubusercontent.com/1701393/152516416-301a5720-467e-4f5b-a9ca-968e2d753343.png)


I only noticed that issue but may be others.

Reply:
Would it be safer to be able to init editor using "uuid and id" or "id only" and gives priority to intenal id on API side to solve your geocat.ch use case ?

We have "uuid and id" because some users have third party app using CSW which offer the possibility to open the editor and CSW does not provide the id information. So not sure which one is better as default but so far "uuid and id" was working fine in most cases.

Another option is to encourage people not using 123 as UUID :)